app/src/main/java/org/sopt/daangnmarket_android/ui/view/main/HomeFragment.kt

```diff
@@ -55,6 +51,44 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun itemNetwork(){
+        val call : Call<ResponseItem> = ServiceCreator.itemService.getItems()
+
+        call.enqueue(object : Callback<ResponseItem> {
+            override fun onResponse(
+                call : Call<ResponseItem>,
+                response: Response<ResponseItem>
+            ){
+                if (response.isSuccessful){
+                    val data = response.body()?.data
+                    if(data != null){
+                        for (i in data){
+                            itemAdapter.itemList.add(
+                                ItemData(
+                                    i.title,
+                                    "청파동",
+                                    i.price,
+                                    i.image,
+                                    i.likeCount,
+                                    i.chatCount,
+                                    i.timeBefore
+                                )
+                            )
+                        }
+                        itemAdapter.notifyDataSetChanged()
```

Comment:
저라면 하나하나 add하지 않고, itemAdapter의 itemList를 val 대신 var로 선언한 뒤 서버로부터 받아온 리스트를 한 번에 할당해버릴 것 같습니다.

```suggestion
itemAdapter.itemList = data
itemAdapter.notifyDataSetChanged()
```

반복문으로 하나하나 넣는 것보다 더 간편할 것 같네요.